Neos.Flow/Classes/Mvc/Routing/Dto/RouteLifetime.php

```diff
@@ -0,0 +1,87 @@
+<?php
+namespace Neos\Flow\Mvc\Routing\Dto;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Annotations as Flow;
+
+/**
+ * RouteLifetime to be associated with matched/resolved routes
+ *
+ * RouteLifetime can be set by Route Part handlers via the ResolveResult/MatchResult return values
+ * If one or more lifetime is added the lifetime will be applied to to the corresponding cache entries.
+ *
+ * @Flow\Proxy(false)
+ */
+final class RouteLifetime
+{
+
+    /**
+     * @var int|null
+     */
+    private $value;
+
+    /**
+     * @param int $value the lifetime
+     */
+    private function __construct(?int $value)
+    {
+        $this->value = $value;
+    }
+
+    /**
+     * Creates an empty instance without a specific lifetime
+     *
+     * @return RouteLifetime
+     */
+    public static function createEmpty(): self
```

Comment:
An "empty" lifetime does not make sense to me. I think we should pull the nullable type up one level

Reply:
true ... will adjust that